context_request.go

```diff
@@ -44,11 +44,12 @@ func NewContextRequest(ctx *Context, log log.Log, validation contractsvalidate.V
 	request := contextRequestPool.Get().(*ContextRequest)
 	httpBody, err := getHttpBody(ctx)
 	if err != nil {
-		log.Error(fmt.Sprintf("%+v", err))
+        	LogFacade.Error(fmt.Sprintf("%+v", errors.Unwrap(err)))
```

Comment:
Don't need to modify this, I optimized the code in a previous PR.

Reply:
Please modify this

Reply:
do I need to return ```log```?
patch-3 of my fork currently uses ```LogFacade```

Reply:
`log` is fine, please update your branch.